.checkpatch.conf

```diff
@@ -29,3 +29,4 @@
 --exclude zboss
 --exclude crypto
 --exclude nrf_security
+--exclude gzll
```

Comment:
Why is gazell added here ?
It's an internal developed code so should be possible to run checkpatch.

Some of the others are because they follow styles outside control of Nodic, nrf_security (ARM mbed), zboss, and openthread.

Reply:
Gazell follows the nRF5 SDK coding style, which is quite different from Zephyr's.

I tried fixing the checkpatch errors. Nevertheless, checkpatch does not accept the 'typedef' usage in GZLL header file. It emits the error message sort of:

<failure message="checkpatch issues" type="failure">No additional types will be considered - file 'scripts/checkpatch/typedefsfile': No such file or directory

Excluding GZLL from checkpatch gets PR compliance checks passed.

Reply:
There is not a plan as of now. Actually the library can still be used in nRF5 SDK with glue code.

Reply:
This is acceptable in this particular case, given that Gazell predates NCS.